sdk/storage/azure-storage-file-share/tests/recordings/test_share_async.test_create_share_async.yaml

```diff
@@ -2,102 +2,28 @@ interactions:
 - request:
```

Comment:
Looks like one pesky recording missed in the revert :)

Reply:
So close.... thanks for pointing it out! :)